core/config/default-config.js

```diff
@@ -541,6 +542,7 @@ const defaultConfig = {
         {id: 'geolocation-on-start', weight: 1, group: 'best-practices-trust-safety'},
         {id: 'notification-on-start', weight: 1, group: 'best-practices-trust-safety'},
         {id: 'csp-xss', weight: 0, group: 'best-practices-trust-safety'},
+        {id: 'has-hsts', weight: 0, group: 'best-practices-trust-safety'},
```

Comment:
Can we do this until documentation is published? This will keep the audit in the JSON result but hide it for anyone looking at the report UI.

```suggestion
        {id: 'has-hsts', weight: 0},
```

Reply:
Hidden now.